nft-client/src/main.rs

```diff
@@ -200,6 +201,26 @@ struct AccountData {
     address:      id::types::AccountAddress,
 }
 
+/// The parameter for the NFT Contract function "CIS2-NFT.mint".
+/// Important: this is specific to this NFT smart contract and contract
+/// functions for minting are not part of the CIS2 specification.
+#[derive(Debug)]
+pub struct MintParams {
+    pub owner:     concordium_contracts_common::Address,
+    pub token_ids: Vec<cis2::TokenIdVec>,
+}
```

Comment:
Since the length of token_ids is assumed to fit into u8, this should be documented at the very least.

Ideally a ::new and ::new_unchecked constructors would be provided.